File: twitter.py
from selenium import webdriver

if __name__ == '__main__':

    # Chrome WebDriver 로 chrome.exe 실행
    browser = webdriver.Chrome('./chromedriver.exe')

    # 사이트 호출
    browser.implicitly_wait(10)
    browser.get("https://twitter.com/login/")

    # 아이디 입력
    browser.implicitly_wait(10)
    id = browser.find_elements_by_name('session[username_or_email]')[0]
    id.send_keys("메일주소")

    # 비밀번호 입력
    browser.implicitly_wait(10)
    pwd = browser.find_elements_by_name('session[password]')[0]
    pwd.send_keys("비밀번호")

    # 트위터 로그인 클릭
    serachbutton = browser.find_element_by_xpath('//*[@id="react-root"]/div/div/div[2]/main/div/div/div[2]/form/div/div[3]/div/div')
    serachbutton.click()



    # 페이지 읽기
    # print(browser.page_source)

    # 드라이브 종료
    browser.quit()

File: golang/twitter.go
// twitter
package main

import (
	"fmt"
	"time"

	"github.com/tebeka/selenium"
)

const (
	path = `c:\GoApp\chromedriver.exe`
	port = 8080
)

func main() {

	opts := []selenium.ServiceOption{}

	server, err := selenium.NewChromeDriverService(path, port, opts...)
	if err != nil {
		fmt.Println(err)
	}
	defer server.Stop()

	c := selenium.Capabilities{"browserName": "chrome"}

	wd, err := selenium.NewRemote(c, fmt.Sprintf("http://localhost:%d/wd/hub", port))
	if err != nil {
		fmt.Println(err)
	}
	defer wd.Quit()

	//사이트 호출
	err = wd.Get("https://twitter.com/login/")
	if err != nil {
		panic(err)
	}

	wd.SetImplicitWaitTimeout(10 * time.Second)

	//아이디 입력
	id, _ := wd.FindElement(selenium.ByName, `session[username_or_email]`)
	id.Clear()
	id.SendKeys("메일주소")

	wd.SetImplicitWaitTimeout(10 * time.Second)

	//비밀번호 입력
	pwd, _ := wd.FindElement(selenium.ByName, `session[password]`)
	pwd.Clear()
	pwd.SendKeys("비밀번호")

	wd.SetImplicitWaitTimeout(10 * time.Second)

	//트위터 로그인 클릭
	serachbutton, _ := wd.FindElement(selenium.ByXPATH, "//div[@data-testid='LoginForm_Login_Button']")
	serachbutton.Click()

	wd.SetImplicitWaitTimeout(10 * time.Second)

	url, _ := wd.CurrentURL()
	title, _ := wd.Title()
	pagesource, _ := wd.PageSource()

	fmt.Println(url)
	fmt.Println(title)
	fmt.Println(pagesource)

	time.Sleep(60 * time.Second)
}
